### ISSUE TITLE
kube-deployment.yml not working: no matches for kind "Deployment" in version "apps/v1beta1"

### ISSUE THREAD

**Issue by sparrowt** [dockersamples/k8s-wordsmith-demo#13]
Following https://www.docker.com/docker-desktop/getting-started-for-windows in step 5 it says to save the contents of [this file](https://raw.githubusercontent.com/dockersamples/k8s-wordsmith-demo/master/kube-deployment.yml) to `kube-deployment.yml` and then run:
```
kubectl apply -f kube-deployment.yml
```
Unfortunately it errors as follows:
```
service/db created
service/words created
service/web created
unable to recognize "kube-deployment.yml": no matches for kind "Deployment" in version "apps/v1beta1"
unable to recognize "kube-deployment.yml": no matches for kind "Deployment" in version "apps/v1beta1"
unable to recognize "kube-deployment.yml": no matches for kind "Deployment" in version "apps/v1beta1"
```

Is the YAML file in this repo wrong, or is the Docker website getting started guide using it wrongly?

### Environment
 - Docker Desktop 2.3.0.5 (48029), 'about' says Kubernetes v1.16.5
 - installed on Win 10 Enterprise 1909
```
kubectl version
Client Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.6-beta.0", GitCommit:"e7f962ba86f4ce7033828210ca3556393c377bcc", GitTreeState:"clean", BuildDate:"2020-01-15T08:26:26Z", GoVersion:"go1.13.5", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.6-beta.0", GitCommit:"e7f962ba86f4ce7033828210ca3556393c377bcc", GitTreeState:"clean", BuildDate:"2020-01-15T08:18:29Z", GoVersion:"go1.13.5", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by ionawright** [comment]
I am also having this problem - I thought I needed to update
`appVersion: apps/v1` 

But still receiving the error for the DB
`service/db unchanged`
`error: error validating "kube-deployment.yml": error validating data: ValidationError(Deployment.spec): missing required field "selector" in io.k8s.api.apps.v1.DeploymentSpec; if you choose to ignore these errors, turn validation off with --validate=false`

**Comment by BretFisher** [comment]
Fixed in #11

**Comment by sparrowt** [comment]
Thanks!
